shopyo/api/cli.py

```diff
@@ -446,32 +446,29 @@ def new(projname, verbose):
     click.echo(f"[x] Project {projname} created successfully!\n")
 
 
-@cli.command("rundebug", with_appcontext=False)
-def rundebug():
-    """runs the shopyo flask app in development mode"""
+def run_helper(mode=""):
+    """helper function"""
     app_path = os.path.join(os.getcwd(), "app.py")
 
     if not os.path.exists(app_path):
         click.secho(f"Unable to find `app.py` in {os.getcwd()}", fg="red")
         sys.exit(1)
 
-    os.environ["FLASK_APP"] = "app:create_app('development')"
-    os.environ["FLASK_ENV"] = "development"
+    os.environ["FLASK_APP"] = "app:create_app('" + mode + "')"
+    os.environ["FLASK_ENV"] = mode
```

Comment:
Might be better to move this helper function to `api/cli_helper.py` and name it `_run_helper` i.e it starts with underscore as it a private function.